server_tests/test/clientTests.js

```diff
@@ -308,7 +317,7 @@ describe("RUST API TEST SUITE", function() {
     });
 
     // CLOSE SESSION
-    describe("Testing /osc/commands/execute camera.closeSession endpoint", function() {
+    describe("Testing /osc/commands/execute camera.closeSession endpoint. Disabled for testing over Wifi", function() {
```

Comment:
I'd probably just remove the title modifiers altogether. Especially the _bubl ones, and shutdown. The wifi ones I'm on the fence but leaning toward just keeping them out.